iroh-dns-server/src/config.rs

```diff
@@ -69,12 +69,19 @@ impl MetricsConfig {
 pub struct MainlineConfig {
     /// Set to true to enable the mainline lookup.
     pub enabled: bool,
+    /// Set custom bootstrap nodes.
+    ///
+    /// If empty this will use the default bittorrent mainline bootstrap nodes as defined by pkarr.
+    pub bootstrap: Vec<SocketAddr>,
 }
 
 #[allow(clippy::derivable_impls)]
 impl Default for MainlineConfig {
     fn default() -> Self {
-        Self { enabled: false }
+        Self {
+            enabled: false,
+            bootstrap: vec![],
```

Comment:
Empty means default? How do you get the default mainline bootstrap nodes?

Reply:
I know it is annoying, but can we make this an enum Default/Custom instead of using empty as "use the defaults"?